src/audio/eq_iir/eq_iir.c

```diff
@@ -522,14 +529,13 @@ static int eq_iir_init_coef(struct sof_eq_iir_config *config,
 		if (s > 0) {
 			size_sum += s;
 		} else {
-			comp_cl_info(&comp_eq_iir, "eq_iir_init_coef(), sections count %d exceeds max",
-				     eq->num_sections);
+			comp_info(mod->dev, "eq_iir_init_coef(), sections count %d exceeds max",
+				  eq->num_sections);
```

Comment:
ditto

Reply:
Fixed too! These are real errors with no way to recover.